src/libs/actions/TaxRate.ts

```diff
@@ -20,6 +31,41 @@ function covertTaxNameToID(name: string) {
     return `id_${name.toUpperCase().replaceAll(' ', '_')}`;
 }
 
+/**
+ * Whether the tax rate can be deleted and disabled
+ */
+function canEditTaxRate(policy: Policy, taxID: string): boolean {
```

Comment:
if a function does not interact with ONYX, let's move it to PolicyUtils.ts. Also help to check other new function here